openfisca_tasks/lint.mk

```diff
@@ -11,7 +11,7 @@ check-syntax-errors: .
 ## Run linters to check for syntax and style errors.
 check-style: $(shell git ls-files "*.py")
 	@$(call print_help,$@:)
-	@flake8 $?
+	@python -m flake8 $?
```

Comment:
Unless I am mistaken and this is really needed for “removing constraint on packages”, this is a [rider](https://en.wikipedia.org/wiki/Rider_(legislation)).
In order to ease review, focus discussion and avoid blocking otherwise useful improvements, please make sure this PR only contains changes that match its description.